packages/block-editor/src/store/private-selectors.js

```diff
@@ -116,7 +117,7 @@ export const getEnabledClientIdsTree = createSelector(
 		state.blockEditingModes,
 		state.settings.templateLock,
 		state.blockListSettings,
-		state.editorMode,
+		__unstableGetEditorMode( state ),
```

Comment:
This seems wrong still?

Reply:
Pushed a commit, does that look better?

Reply:
Looks like some tests need to be updated now.

Reply:
Should now be fixed too.

Reply:
Thanks for the fixes, @talldan! This was an end-of-day PR for me, and I didn't wait for the tests...

Reply:
No worries, thought I'd help out on the night shift, as it looked fairly urgent 😄